package ExceptionHandling;

public class Arithmetic {
	public static void main(String[] args) {
		double div = 10 / 0;
		System.out.println("Double: " + div);

		try {

		} catch (Exception e) {
			// TODO: handle exception
		}

	}
}
